### PR TITLE
Don't use EnforceRange types for flags consts on interfaces

### PR DESCRIPTION
**Pull request by kainino0x**
This is an "output" field, where (at least in principle), EnforceRange is not allowed.

While I don't think WebIDL ever officially added this requirement, it is almost certainly supposed to be there. And apparently Chromium's IDL generators enforce it.

Issue: #549

***
<a href="https://pr-preview.s3.amazonaws.com/kainino0x/gpuweb/pull/928.html" title="Last updated on Jul 20, 2020, 11:09 PM UTC (19437b2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/gpuweb/gpuweb/928/184f68b...kainino0x:19437b2.html" title="Last updated on Jul 20, 2020, 11:09 PM UTC (19437b2)">Diff</a>

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -1355,16 +1355,16 @@ dictionary GPUBufferDescriptor : GPUObjectDescriptorBase {
...
-    const GPUBufferUsageFlags MAP_WRITE     = 0x0002;
-    const GPUBufferUsageFlags COPY_SRC      = 0x0004;
-    const GPUBufferUsageFlags COPY_DST      = 0x0008;
-    const GPUBufferUsageFlags INDEX         = 0x0010;
-    const GPUBufferUsageFlags VERTEX        = 0x0020;
-    const GPUBufferUsageFlags UNIFORM       = 0x0040;
-    const GPUBufferUsageFlags STORAGE       = 0x0080;
-    const GPUBufferUsageFlags INDIRECT      = 0x0100;
-    const GPUBufferUsageFlags QUERY_RESOLVE = 0x0200;
+    const unsigned long MAP_READ      = 0x0001;
```
Comment (kvark) [inline spec/index.bs:1358]: that's super sad: we are making a typedef just so we do *not* use it :(
should we at least make `typedef unsigned long GPUFlags` to use there?

Comment (kainino0x) [reply spec/index.bs:1358]: The typedef mainly exists to be used in faraway input slots like GPUBufferDescriptor. Your proposed typedef would be fine with me though.

Comment (kainino0x) [reply spec/index.bs]: Done

**Comment by kainino0x** [comment]
Editors' meeting (+@litherum): this change is only really needed if the upstream WebIDL issue resolves to disallow [EnforceRange] on `const`s. We should get it resolved upstream in https://github.com/heycam/webidl/issues/842

**Comment by kainino0x** [comment]
> Consider it resolved!

-- https://github.com/heycam/webidl/issues/842#issuecomment-668258877

So we should go ahead with a change here.

**Comment by kainino0x** [comment]
Just going to land this and we can make more changes later if we want to.

**Pull request merged**